tests/cross_functional/kcs/test_selinux_relabel_solution.py

```diff
@@ -300,15 +301,19 @@ def test_selinux_relabel_for_existing_pvc(
         self.apply_selinux_solution_on_existing_pvc(self.pvc_obj)
 
         # Delete pod so that fix will be applied for new pod
-        self.pod_obj = self.get_app_pod_obj()
-        self.pod_obj.delete(wait=True)
-        self.pod_obj = self.get_app_pod_obj()
-        assert wait_for_pods_to_be_running(
-            pod_names=[self.pod_obj.name], timeout=600, sleep=15
-        ), f"Pod {self.pod_obj.name} didn't reach to running state"
+        for _ in range(2):
+            self.pod_obj = self.get_app_pod_obj()
+            self.pod_obj.delete(wait=True)
```

Comment:
since it is a deploymentconfig application, ideally it should be deleted by scaling down the deploymentconfig.
Please replace deletion of pod with deployment scaledown and up